alluxio-python-library/alluxio/alluxio_file_system.py

```diff
@@ -156,6 +166,111 @@ def listdir(self, path):
                 f"Error when listing path {path}: error {e}"
             ) from e
 
+    def load(
+        self,
+        file_path,
+        timeout=None,
+    ):
+        """
+        Loads a file.
+
+        Args:
+            file_path (str): The full ufs file path to load data from
+            timout (integer): The number of seconds for timeout
+
+        Returns:
+            result (boolean): Whether the file has been loaded successfully
+        """
+        worker_host = self._get_preferred_worker_host(file_path)
+        return self._load_file(worker_host, file_path, timeout)
+
+    def submit_load(
+        self,
+        file_path,
+    ):
+        """
+        Submits a load job for a file.
+
+        Args:
+            file_path (str): The full ufs file path to load data from
+
+        Returns:
+            result (boolean): Whether the job has been submitted successfully
+        """
+        worker_host = self._get_preferred_worker_host(file_path)
+        try:
+            response = self.session.get(
+                self.LOAD_SUBMIT_URL_FORMAT.format(
+                    worker_host=worker_host,
+                    http_port=self.http_port,
+                    path=file_path,
+                ),
+            )
+            response.raise_for_status()
+            return b"successfully" in response.content
```

Comment:
Not related to this PR, but could you give our RESTful load API some advice on what return values are good for python API?

Reply:
This process is error prone, e.g. what if they change the reply message

Reply:
Confirmed with China team, it would be hard for us to have a true or false return from the RESTFUL load submit API, let's keep it this way.